examples/modules/fetch-newrelic-api-access-key/main.tf

```diff
@@ -0,0 +1,41 @@
+
+provider "graphql" {
+  url = var.fetch_access_keys_service.graphiql_url
+  headers = {
+    "Content-Type" = "application/json"
+    "API-Key" = var.fetch_access_keys_service.api_key != "" ? var.fetch_access_keys_service.api_key : var.create_access_keys_service.api_key
+  }
+}
+
+data "graphql_query" "basic_query" {
+  query_variables = {
+    "id"        = var.fetch_access_keys_service.key_id
+    "key_type"  = var.fetch_access_keys_service.key_type
+  }
+  query  = file("query.gql")
+  count = local.is_resource_created ? 0 : 1
+}
+
+resource "newrelic_api_access_key" "api_access_key" {
+  count  = var.create_access_keys_service.newrelic_account_id != "" ? 1 : 0
+  account_id  = var.create_access_keys_service.newrelic_account_id
+  key_type    = var.create_access_keys_service.key_type
+  ingest_type = var.create_access_keys_service.ingest_type
+  name        = "APM ${var.create_access_keys_service.key_type} License Key for ${var.create_access_keys_service.name}"
```

Comment:
@vinay-newrelic this statement needs a change as the `ingest_type` can be `BROWSER` or `INGEST` and this ingest type would need to be printed accordingly. In addition, it might also be useful to add the ingest type to the output attributes printed.

Reply:
@pranav-new-relic i have added ingest type also as an output variable but the ingest type key would ouput null in case of USER keys ...I tried to make it a conditional output but did not find an appropriate way .